packages/docs/docs/fhir-datastore/patient-deduplication/architecture-overview.md

```diff
@@ -0,0 +1,37 @@
+# Architecture Overview
+
+## Terminology
```

Comment:
Unfortunately "Terminology" means something specific in FHIR.  Can we change this to "Establishing shared vocabulary", also open to other words but not Terminology as [FHIR Terminology](https://build.fhir.org/terminology-service.html#:~:text=A%20FHIR%20terminology%20service%20is,inherently%20known%20terminologies%20providing%20support.) is something in and of itself.

Reply:
This has been changed in the lates commit